static/search.js

```diff
@@ -740,5 +728,43 @@ var Search = Control.extend({
 
 });
 
+function getLocalStorageItem(key){
+	if(!window.localStorage){
+		return null;
+	}
+
+	var storageItem = localStorage.getItem(key);
+
+	if(storageItem){
+		return JSON.parse(storageItem);	
+	}
+
+	return null;
+}
+
+function setLocalStorageItem(key, data){
+	if(!window.localStorage){
+		return null;
+	}
+	if(data){
+		localStorage.setItem(key, JSON.stringify(data));
+		return true;
+	}
+	return null;
+}
+
+function localStorageIsAvailable(){
+	var t = new Date().getTime();
```

Comment:
I think this could be made simpler by using a constant key (something like `local-storage-test` or `localStorageKeyPrefix('test')`).